Microsoft/WebTemplateStudio
# New Splash SVGs for 'New Project' and 'Summary Page'

**How To Test**
1) Check light, dark, high contrast themes
2) Check that the splash SVGs resizes under the text/cards/input fields/buttons (it should act like a background)

![darknewproject](https://user-images.githubusercontent.com/21695878/61500149-c5bd4600-a97e-11e9-84ba-ff098905bb89.PNG)
![lightsummary](https://user-images.githubusercontent.com/21695878/61500150-c5bd4600-a97e-11e9-9544-5fd0732703ef.PNG)
